java/org.travlyn.server/src/test/java/org/travlyn/server/externalapi/access/DBpediaAccessTest.java

```diff
@@ -1,19 +1,41 @@
 package org.travlyn.server.externalapi.access;
 
+import org.junit.After;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Tag;
+import org.travlyn.server.ApiTest;
 import org.travlyn.shared.model.api.City;
 
+import java.io.IOException;
+
 @Tag("unit")
-public class DBpediaAccessTest {
+public class DBpediaAccessTest extends ApiTest {
+
+    @Before
+    @BeforeEach
+    public void setUp() throws Exception {
```

Comment:
Should we really mock this test? It does exactly one call to dbpedia... I think if we mock this a "Access test" wouldnt be that sensefull...